xmlvalidate/cmd/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/artefactual-sdps/temporal-activities/xmlvalidate"
+)
+
+func usage() {
+	fmt.Print(`
+USAGE: xmlvalidate [options] XMLFILE
+
+XMLFILE  Path of the XML file to be validated
+
+OPTIONS:
+  --xsd PATH  Path of an XSD file to validate against
```

Comment:
I made it an option for two reasons:

1. It avoids confusing the order of the XML file path and XSD file path, e.g. `xmlvalidate person.xsd people.xml` vs. `xmlvalidate person.xsd people.xml`
2. It leaves open the option of adding a `--dtd` option to validate XML against a DTD file instead of an XSD

I think the first reason justifies typing the extra `--xsd` .